RELEASING.md

```diff
@@ -0,0 +1,36 @@
+# Releasing Guide
+
+AX releasing workflow is semi-guided by the CI logic comprised of [GitHub actions scripts](.github/workflows/), [makefile](./Makefile), and [cosmos-release tool](./rust/release/).
+
+## General Release Steps
+
+> - Note: Explanations for the text in bold can be found in the [glossary section](#glossary).
+
+1. A release starts when a **changelog commit** is created.
+1. The previous step triggers the creation of a **Release PR**. This PR will initially fail because the version validation detects that a version bump is required.
+1. _[Manually]_ With a developer's assistance, each impacted **product's** version is bumped correctly. These product's version can be found in each of the corresponding cargo.toml or package.json file.
+1. _[Manually]_ the **Release PR** is merged when its build and validation chain is successful.
+1. After **Release PR** is merged to the `master` branch, a publication process will run; it publishes all **products** listed on the previously merged release.
+1. _[Manually]_ A developer publishes `rust` crates that have been changed.
+
+## Glossary:
+
+- **`Actyx` Product**: or simply product, refers to publishable artifacts from this repository (`ax`, `ax-core`, `node-manager`, etc).
+- **changelog commit**: a conventional commit with a valid product designation that will trigger a release process and will be included in the release changelog.
```

Comment:
This line should describe exactly how a changelog is attached to a commit instead of the consequences this will have down the line. It should probably point to a section below detailing the changelog entry syntax.